tests/functional/metadata/mongodb/listObject.spec.js

```diff
@@ -0,0 +1,306 @@
+            cbIterator => {
+                count++;
+                // eslint-disable-next-line
+                return metadata.putObjectMD(bucketName, objName, objVal, params,
+                                        logger, cbIterator);
+            },
+            err => {
+                if (err) {
+                    return cb(err);
+                }
+                return cb(null);
+            },
+        );
+    }
+
+    beforeAll(done => {
+        mongoserver.waitUntilRunning().then(() => {
+            const opts = {
+                mongodb: {
+                    replicaSetHosts: 'localhost:27020',
+                    writeConcern: 'majority',
+                    replicaSet: 'rs0',
+                    readPreference: 'primary',
+                    database: DB_NAME,
+                },
+            };
+            metadata = new MetadataWrapper(IMPL_NAME, opts, null, logger);
+            metadata.setup(done);
+        });
+    });
+
+    afterAll(done => {
+        async.series([
+            next => metadata.close(next),
+            next => mongoserver.stop()
+                .then(() => next())
+                .catch(next),
+        ], done);
+    });
+
+    beforeEach(done => {
+        const bucketMD = BucketInfo.fromObj({
+            _name: BUCKET_NAME,
+            _owner: 'testowner',
+            _ownerDisplayName: 'testdisplayname',
+            _creationDate: new Date().toJSON(),
+            _acl: {
+                Canned: 'private',
+                FULL_CONTROL: [],
+                WRITE: [],
+                WRITE_ACP: [],
+                READ: [],
+                READ_ACP: [],
+            },
+            _mdBucketModelVersion: 10,
+            _transient: false,
+            _deleted: false,
+            _serverSideEncryption: null,
+            _versioningConfiguration: null,
+            _locationConstraint: 'us-east-1',
+            _readLocationConstraint: null,
+            _cors: null,
+            _replicationConfiguration: null,
+            _lifecycleConfiguration: null,
+            _uid: '',
+            _isNFS: null,
+            ingestion: null,
+        });
+        const versionParams = {
+            versioning: true,
+            versionId: null,
+            repairMaster: null,
+        };
+        async.series([
+            next => metadata.createBucket(BUCKET_NAME, bucketMD, logger, err => {
+                if (err) {
+                    return next(err);
+                }
+                return next();
+            }),
+            next => {
+                const params = {
+                    objName: 'pfx1-test-object',
+                    objVal: {
+                        key: 'pfx1-test-object',
+                        versionId: 'null',
+                    },
+                    nbVersions: 5,
+                };
+                putBulkObjectVersions(BUCKET_NAME, params.objName, params.objVal, versionParams,
+                params.nbVersions, next);
```

Comment:
i don't see the point of having this structure: just inline the member to keep the code simpler & cleaner

```suggestion
                const objVal = {
                    key: 'pfx1-test-object',
                    versionId: 'null',
                };
                putBulkObjectVersions(BUCKET_NAME, objVal.key, objVal, versionParams, 5, next);
```

Reply:
fixed in https://github.com/scality/Arsenal/pull/1729/commits/a6f48b9f342344e6df5ca2e3b7a2d03d8737fbba